test/test_link_has_href.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
```

Comment:
Should this be in `test/accessibility/`?

Reply:
I didn’t put the tests inside a subfolder because `bundle exec rake` doesn’t find them. It only executes the test on the test folder, no subfolders